packages/admin-api/.editorconfig

```diff
@@ -0,0 +1,26 @@
+# http://editorconfig.org
```

Comment:
Can you please manually delete this file? 

I have already pushed a bug fix - it shouldn't get generated again. Sorry!

Reply:
sure thing! :relaxed: